.snippets/code/develop/blockchains/testing/setup/genesis-config.rs

```diff
@@ -0,0 +1,11 @@
+pub mod integration_testing {
```

Comment:
This file is actually being compiled? what version of `polkadot-sdk` is it using?

Reply:
@kianenigma  Yes, this code has been compiled. I took inspiration from here for this page: https://github.com/AstarNetwork/Astar/tree/master/tests/integration

And then, my implementation is here, using the `solochain-template`: https://github.com/nhussein11/solochain-template/blob/nhussein11/integration-testing/runtime/src/integration_testing.rs

Reply:
But the point is not that it has been compiled "_once_", it is to have a CI that continuously does it. 

If you want to copy something that was once compiled, I think we should at least be strict on the polkadot-sdk stable version; Only use code that you know was correct at a particular stable version, and ensure all code snippets here are correct in that version.